lib/recipients/project.js

```diff
@@ -133,7 +132,8 @@ module.exports = async ({ schema, logger, task }) => {
       emailTemplate: 'project-expiring',
       addTaskTypeToSubject: false,
       applicant: project.licenceHolder,
-      logMsg: `project expiring in ${months} months`
+      logMsg: `project expiring in ${months} months`,
+      identifier: `${project.id}-${action}`
```

Comment:
What is `action` in this context? It might be safer to set to `${project.id}-expiring-${months}`